pkgs/stdenv/generic/setup.sh

```diff
@@ -206,8 +206,8 @@ addToSearchPath() {
 # syntax when they switch to setting __structuredAttrs = true.
 prependToVar() {
     local -n nameref="$1"
+    local useArray type
```

Comment:
What does the type do?

Reply:
My intent is to make two local variables called `useArray` and `type`. I think I have the syntax correct.

Reply:
Ah yes, forgot there was a variable called type

Reply:
Syntax highlighting thinks this is the type builtin